<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Upload_Dialog</class>
 <widget class="QDialog" name="Upload_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TurnFix - Ergebnisse auf turnfix.de laden</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../../resources/TurnFix.qrc">
    <normaloff>:/appicons/icon.png</normaloff>:/appicons/icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="chk_rnd">
     <property name="text">
      <string>Rundengesamtwertung anstatt Wettkampfwertung hochladen</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="chk_aw">
     <property name="text">
      <string>Ausgangswerte in Ergebnisliste</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Details anzeigen:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="cmb_detail">
     <item>
      <property name="text">
       <string>keine Details</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Turnkreis/-gau</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Landesverband</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Land</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="bbx_done">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QProgressBar" name="pbar">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="lbl_progress">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="chk_format">
     <property name="text">
      <string>PDF-Dateien im Querformat</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../resources/TurnFix.qrc"/>
 </resources>
 <connections/>
</ui>
